fdbrpc/QueueModel.h

```diff
@@ -27,13 +27,35 @@
 #include "flow/Knobs.h"
 #include "flow/ActorCollection.h"
 
-
+/*
+The data structure used for the client-side load balancer to decide which
+storage server to read data from. Conceptually, it represents the size of
+storage server read request queue. One QueueData represents one storage
```

Comment:
IIUC, it is the SS's write queue size. https://github.com/apple/foundationdb/blob/ef0bf2728e684fa1280af32b959c821c6dfa5ef4/fdbserver/storageserver.actor.cpp#L743-L746 

@sfc-gh-etschannen could you please confirm?

Reply:
I asked @sfc-gh-etschannen earlier and got more clarification. The smoothOutstanding is a sole client-side metric representing the number of outstanding the "local" client sent to each storage server.

This queueSize() you pointed out is not sent back to the client directly, but may influence the penalty computation.